lib/api/manipulation.js

```diff
@@ -363,6 +363,47 @@ exports.wrapInner = _wrap(function (el, elInsertLocation, wrapperDom) {
   updateDOM(wrapperDom, el);
 });
 
+/**
+ * The .unwrap() function, removes the parents of the set of matched elements
+ * from the DOM, leaving the matched elements in their place.
+ *
+ * @example <caption>without selector</caption>
+ *   const $ = cheerio.load(
+ *     '<div id=test>\n  <p>Hello</p></div>\n  <div><p>World</p>\n</div>'
+ *   );
+ *   $('#test p').unwrap();
+ *
+ *   //=> <div id=test>
+ *   //     <p>Hello</p>
+ *   //     <p>World</p>
+ *   //   </div>
+ *
+ * @example <caption>with selector</caption>
+ *   const $ = cheerio.load(
+ *     '<div id=test>\n  <p>Hello</p>\n  <b><p>World</p></b>\n</div>'
+ *   );
+ *   $('#test p').unwrap('b');
+ *
+ *   //=> <div id=test>
+ *   //     <p>Hello</p>
+ *   //     <p>World</p>
+ *   //   </div>
+ *
+ * @param {string} selector - A selector to check the parent element against. If
```

Comment:
Mark this as optional?

```suggestion
 * @param {string} [selector] - A selector to check the parent element against. If
```